### PR TITLE
chore(site-showcase): add GVD Renovations site

### PR DESCRIPTION
**Pull request by dkrasniy**
Added GVD Renovations Gatsby site to showcase!

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -4431,3 +4431,15 @@
     - Featured
   built_by: Algolia
   featured: true
+- title: GVD Renovations
+  url: https://www.gvdrenovations.com/
+  main_url: https://www.gvdrenovations.com/
+  description: >
+    GVD Renovations is a home improvement contractor with a well know reputation as professional, quality contractor in California.
```
Comment (DSchau) [inline docs/sites.yml]:
```suggestion
    GVD Renovations is a home improvement contractor with a well known reputation as a professional, quality contractor in California.
```

File: docs/sites.yml [hunk 2]
```diff
@@ -4431,3 +4431,15 @@
...
+  categories:
+    - Home Improvement
```
Comment (DSchau) [inline docs/sites.yml]: Could you do me a quick favor and ensure that these categories are more generally applicable?

In other words--if you check out https://www.gatsbyjs.org/showcase/, these categories should have appeared before if at all possible! We want to keep these as "general" as possible, so in this case I'd recommend something like:

```yaml
categories:
  - Business
  - Real Estate
```

and then any other that you think fit! Please let me know if you have any questions on this!

File: docs/sites.yml (hunk 1 above)
Comment (dkrasniy) [reply docs/sites.yml]: Sure! Thanks for catching that

File: docs/sites.yml (hunk 2 above)
Comment (dkrasniy) [reply docs/sites.yml]: I see. Let's go with 

- Construction
- Business

Comment (DSchau) [reply docs/sites.yml]: Works for me!

**Comment by dkrasniy** [comment]
Committed the two changes. Let me know if I need to do anything else. Thanks for the quick turnaround!

**Comment by DSchau** [comment]
Thanks for using Gatsby! Merging this in!

**Pull request merged**